core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -1779,6 +1779,11 @@ def _train_and_save(
                     return model_names_trained
                 if self._time_limit is not None and self._time_train_start is not None:
                     time_left_total = self._time_limit - (fit_start_time - self._time_train_start)
+                    # If only a very small amount of time remains, skip training
+                    min_time_required = min(self._time_limit * 0.01, 10)
```

Comment:
Not sure about the intended logic, but should this be `max` instead?

Reply:
`min` on purpose, as if a user gives 10 seconds for runtime, we should try to train the models with 3 seconds left, but if given 10000 seconds, probably skip the model if 3 seconds are left.